packages/salesforcedx-vscode-core/test/vscode-integration/util/orgUtil.test.ts

```diff
@@ -70,15 +70,15 @@ describe('orgUtil tests', () => {
         }
       ]);
       const orgList = new OrgList();
-      await checkForExpiredOrgs(orgList);
+      await checkForSoonToBeExpiredOrgs(orgList);
 
       expect(showWarningMessageSpy.called).to.equal(false);
       expect(appendLineSpy.called).to.equal(false);
       expect(showChannelOutputSpy.called).to.equal(false);
-      expect(authInfoCreateStub.called).to.equal(false);
+      expect(authInfoCreateStub.called).to.equal(true);
```

Comment:
Why is this changed from false to true?  I see that Line 90 of OrgUtil.test.ts (the unit test for only dev hubs) seems to be a similar case to this line, and it says `expect(authInfoCreateSpy).not.toHaveBeenCalled();`

Reply:
It should have been called, but I guess this file is outdated. I'll delete it since I covered all the cases present in the jest test file.